### DISCUSSION TITLE
Web UI?

### DISCUSSION THREAD

**Discussion by diamant-x** [hassio-addons/addon-wireguard#126] [Ideas]
First of all, thanks for the work done with this add-on. I use it daily driving it to avoid exposing my hassio instance to the net so that wireguard handles all the wan to lan traffic. Thanks!

While this tool is easy enough to be used as is, as a mean to become even more complete, maybe it could be extended with a web UI such as the one here https://github.com/WeeJeWel/wg-easy/ ?
It might collide with the text-based config approach to setup new clients and setting, but might provide more amicable operations?

**Comment by frenck** [comment]
That project is to young to take serious at this point. Maybe in the future, who knows.
